src/components/content-header-card/content-header-card.module.css

```diff
@@ -0,0 +1,174 @@
+.contentHeaderCard {
+	padding: 0;
+
+	& .cardTop {
+		padding: 16px;
+		display: flex;
+
+		& .icon {
+			margin-right: 12px;
+			display: none;
+
+			@media (--dev-dot-tablet-up) {
+				display: block;
+			}
+		}
+
+		& .content {
+			flex-grow: 1;
+
+			& .upper {
+				display: flex;
+				gap: 16px;
+				flex-direction: column-reverse;
+				@media (--dev-dot-tablet-up) {
+					flex-direction: row;
+				}
+
+				& .left {
+					flex-grow: 1;
+
+					& .titleWrap {
+						display: flex;
+						flex-wrap: wrap;
+						align-items: baseline;
+						@media (--dev-dot-tablet-up) {
+							flex-wrap: unset;
+						}
+
+						& .title {
+							color: var(--token-color-foreground-strong);
+						}
+
+						& .attribution {
+							color: var(--token-color-foreground-faint);
+							margin-left: 4px;
+						}
+					}
+
+					& .description {
+						margin-top: 2px;
+						color: var(--token-color-foreground-primary);
+					}
+				}
+
+				& .dropdown {
+					flex-shrink: 0;
+					width: 100%;
+					& > button {
+						width: 100%;
+					}
+
+					@media (--dev-dot-tablet-up) {
+						width: auto;
+						& > button {
+							width: auto;
+						}
+					}
+				}
+			}
+		}
+
+		& .lower {
+			display: flex;
+			flex-direction: column;
+			align-items: flex-start;
+			@media (--dev-dot-tablet-up) {
+				flex-direction: row;
+				align-items: flex-end;
+			}
+
+			& .badgeList {
+				margin-top: 16px;
+			}
+
+			& .note {
+				color: var(--token-color-foreground-faint);
+				flex-grow: 1;
+				text-align: right;
+				margin-top: 12px;
+				@media (--dev-dot-tablet-up) {
+					margin-top: 6px;
+				}
+			}
+		}
+	}
+
+	& .cardBottom {
+		padding: 12px 16px;
+		background: var(--token-color-surface-faint);
+
+		/* This is the color from --token-surface-base-box-shadow,
+		which the color value from it is not in a variable alone */
+		border-top: 1px solid rgba(101, 106, 118, 0.2);
+		border-radius: 0 0 6px 6px;
+		display: flex;
+		flex-direction: column;
+		align-items: flex-start;
+		gap: 16px;
+
+		@media (--dev-dot-tablet-up) {
+			flex-direction: row;
+			justify-content: space-between;
+			align-items: center;
+		}
+
+		& .linksList {
+			list-style: none;
+			display: flex;
+			padding: 0;
+			margin: 0;
+			& li {
+				&:not(:first-of-type) {
+					margin-left: 12px;
+				}
+				& > * {
+					/* TODO: Rather than override our existing link component, 
+					we should eject or build in support for what we're trying
+					to do here. These styles assume the StandaloneLink styles
+					are part of its contract, but they are not and are subject
+					to change */
+					text-decoration: underline;
+					color: var(--token-color-foreground-primary);
+					&:hover {
+						color: var(--token-color-foreground-strong);
+					}
+				}
+			}
+		}
+
+		& .buttonsList {
+			padding: 0;
+			margin: 0;
+			list-style: none;
+			display: flex;
+			flex-grow: 1;
+			flex-direction: row;
+			@media (--dev-dot-tablet-up) {
+				flex-direction: row-reverse;
```

Comment:
These are reversed on desktop. Generally, CTA button alignment are aligned in the direction of button list alignment (relative to its container).  e.g. a right aligned button list would have its primary button to the right.

<img width="806" alt="CleanShot 2023-03-03 at 14 49 57@2x" src="https://user-images.githubusercontent.com/2105067/222847540-f332110b-53ae-4120-9794-b52ff5dabf7c.png">

<img width="405" alt="CleanShot 2023-03-03 at 14 50 03@2x" src="https://user-images.githubusercontent.com/2105067/222848291-35471c5a-7b91-4364-9e25-07963a5923f7.png">

I believe that we additionally would want the tabbing order to match this, as the primary button should go first. The text box of this comment actually illustrates the same behavior:

https://user-images.githubusercontent.com/2105067/222849715-26ef023f-66f3-46db-9008-e447f38222fb.mp4

Reply:
The [Figma designs](https://www.figma.com/file/VD7ahvXuXWJApeGnhbW4hv/Developer?node-id=20364%3A254758&t=7oII6fQE1NXSOf75-11) I'm looking at don't show the button order changing on narrower viewports.

![image](https://user-images.githubusercontent.com/43934258/223172708-9932bfc6-8b3b-460e-afe3-81191f5bcc64.png)

The visual order of these elements must also match the DOM order, and on wider viewports they don't match. This needs to be fixed before merging this PR.

https://user-images.githubusercontent.com/43934258/223172986-30e955e4-fd70-49e4-9814-36c83cf1da13.mov

Reply:
I had a conversation with @braydenlove to verify this behavior via DM.

Reply:
For the visual order changing -- that sounds good and glad y'all are aligned! We'll just need to make the DOM order match the visual order. It falls under [SC 1.3.2: Meaningful Sequence](https://www.w3.org/WAI/WCAG21/Understanding/meaningful-sequence) which is a Level A SC.

Reply:
Sorry I haven't had a chance to update the designs after we pairing on this @BrandonRomano. After chatting we decided on keeping the primary button on the side the group is floating (if button group is floating right primary is displayed on the right, if button group is floating left primary is displayed on the left). Here are a few scenarios: https://www.figma.com/file/VD7ahvXuXWJApeGnhbW4hv/Developer?node-id=20364%3A254758&t=KRTf0BF6CQAWZNIQ-1

Reply:
Would the definition of "meaningful sequence" in [SC 1.3.2: Meaningful Sequence](https://www.w3.org/WAI/WCAG21/Understanding/meaningful-sequence) come into play here?

> A sequence is meaningful if the order of content in the sequence cannot be changed without affecting its meaning.

It seems possible we can change the order of these action buttons without affecting the meaning of the content on the page. As an example, I think this is why the GitHub commenting buttons example that Brandon mentioned are likely acceptable:

![CleanShot 2023-03-06 at 16 19 43](https://user-images.githubusercontent.com/4624598/223233404-4238f45f-7923-491e-8f16-041a2065c16e.gif)

It seems like the visual and DOM order there doesn't technically match - I'm reading in a left-to-right language, and visually the buttons appear `Cancel` and then `Comment` left-to-right, but the focus order is `Comment` then `Cancel`. But, I think this is okay because the visual order isn't affecting the meaning. And, I think there is a case to be made that with the way we scan content, having the primary button at the "edge" of the container (but still first in DOM order!) is likely the most all-around intuitive solution.

Reply:
I do think we imply some meaning here with 2 visual specifics:

- the right-alignment of the actions group
- the rightmost action having a color that stands out from the others

Something important to consider with the ordering of actions like this whether or not users are aware of all their options.

- Sighted users that don't rely on assistive technology or things like magnification are able to visually scan a page and see all of their options at once.
- Many screen reader users don't have the ability to view a page at-a-glance. They experience a page linearly. So if we place a primary action first in a group of buttons, they may not know what all of their options are.
- Users that can see the focus indicator move as they navigate a page, whether with assistive technology or not, can easily become disoriented, confused, distracted, etc. if the page jumps around in a way they don't expect.

The Understanding docs for [Technique C27: Making the DOM order match the visual order](https://www.w3.org/WAI/WCAG21/Techniques/css/C27) get into that last point more.

> The objective of this technique is to ensure that the order of content in the source code is the same as the visual presentation of the content. The order of content in the source code can be changed by the author to any number of visual presentations with CSS. Each order may be meaningful in itself but may cause confusion for assistive technology users. This could be due to the user switching off the author-specified presentation, by accessing the content directly from the source code (such as with a screen reader), or by interacting with the content with a keyboard. If a blind user, who reads the page with a screen reader that follows the source order, is working with a sighted user who reads the page in visual order, they may be confused when they encounter information in different orders. A user with low vision who uses a screen magnifier in combination with a screen reader may be confused when the reading order appears to skip around on the screen. A keyboard user may have trouble predicting where focus will go next when the source order does not match the visual order.
>
> There may also be situations where the visually presented order is necessary to the overall understanding of the page, and if the source order is presented differently, it may be much more difficult to understand.
>
> When the source order matches the visual order, everyone will read the content and interact with it in the same (correct) order.